src/rustup-cli/rustup_mode.rs

```diff
@@ -788,6 +792,14 @@ fn doc(cfg: &Cfg, m: &ArgMatches) -> Result<()> {
         "index.html"
     };
 
+    if m.is_present("print-only") {
+        let path =  toolchain.doc_path(doc_url);
+        match path {
+            Ok(v) => return Ok(println!("file:/{}", v.to_str().unwrap())),
```

Comment:
This will produce a URL like `file:/C:\Foo\bar` on windows, or possibly something even stranger, can you update this to work on windows, and also add a test for this command.

Reply:
Sure, I'll try to work it out.

Reply:
BTW, shouldn't the URL be just the URL with no platform-specific prefixes? 
I mean just the 'C:\Foo\bar' or whatever the platform-specific URL is.

Reply:
That is probably fine - at least on windows the browser is smart enough to turn a path into a `file:///` URL.